app/controllers/orders/detail.js

```diff
@@ -6,22 +6,13 @@ import AjaxPromise from 'browse/utils/ajax-promise';
 export default applicationController.extend({
   order: Ember.computed.alias('model.order'),
   showCancelBookingPopUp: false,
-  orderSummaryTabs: [{
```

Comment:
isn't that needed for the tabs ? 😱

Reply:
nope, not needed now, code has been changed.

Reply:
cool :)